budgeteer-web-interface/src/main/java/org/wickedsource/budgeteer/web/pages/budgets/RemainingBudgetFilterModel.java

```diff
@@ -0,0 +1,25 @@
+package org.wickedsource.budgeteer.web.pages.budgets;
+
+import org.apache.wicket.injection.Injector;
+import org.apache.wicket.model.LoadableDetachableModel;
+import org.apache.wicket.spring.injection.annot.SpringBean;
+import org.wickedsource.budgeteer.service.budget.BudgetService;
+import org.wickedsource.budgeteer.web.BudgeteerSession;
+
+public class RemainingBudgetFilterModel extends LoadableDetachableModel<Long> {
```

Comment:
Above, you used the term "BudgetRemaining", here you use "RemainingBudget". Shouldn't this be consistent?

Reply:
Also fixed